fgpyo/util/logging.py

```diff
@@ -151,6 +151,26 @@ def record(
         else:
             return False
 
+    def record_alignment(
+        self,
+        rec: AlignedSegment,
+    ) -> bool:
+        """Correctly record pysam.AlignedSegments (zero-based coordinates).
+
+        Args:
+            rec: pysam.AlignedSegment object
+
+        Returns:
+            true if a message was logged, false otherwise
+        """
+        if rec.reference_start is None:
+            position_from_rec = 0
+        else:
+            position_from_rec = rec.reference_start + 1
+        reference_name_from_rec = rec.reference_name
+        logged = self.record(position=position_from_rec, reference_name=reference_name_from_rec)
+        return logged
```

Comment:
I think this would likely be clearer as:

```suggestion
        if rec.reference_start is None:
            return self.record(None, None)
        else:
            return self.record(rec.reference_name, rec.reference_start + 1)
```